spec/rails_helper.rb

```diff
@@ -1,6 +1,6 @@
 # frozen_string_literal: true
 
-ENV['RAILS_ENV'] ||= 'test'
+ENV['RAILS_ENV'] = 'test'
```

Comment:
📛📛📛

Reply:
What does that emoji mean? :-)

Reply:
I've now triple-badge-verified your change.

_(But note again, this has zero weight or impact)_

Reply:
Would you be willing to receive a quadruple badge if I can create one ... or do you think three is sufficient?

Reply:
Three is enough, I think. But an animated GIF would be nice.